README.md

```diff
@@ -170,17 +180,19 @@ to customize them with your credentials.
     cp -R ~/compute-video-demo-salt/srv/* /srv
     ```
 
-1. Now you will need to get a copy of your private Service Account key
-uploaded to your `salt` master in the proper format.  Once uploaded, you can
-convert the Service Account private key file from the PKCS12 format to the
-RSA/PEM file format.  You can do that with the `openssl` utility,
+1. Next, you'll set up your private Service Account key on the Salt master.
+  1. Upload a copy of your private Service Account key to your `salt` master (this is the file we downloaded earlier).  You may do this with [scp](https://cloud.google.com/compute/docs/instances/transfer-files#scp), 
+  or by using gcloud's copy-files command, like so:
+   ```
+   gcloud compute copy-files /path/to/project-id-0375632f3fe4.p12 salt:~ --zone us-central1-b
+   ```
+  1. Once uploaded, you can convert the Service Account private key file from the PKCS12 format to the
+RSA/PEM file format.  You can do that with the `openssl` utility.  On the 'salt' master, as root, run,
     ```
-    openssl pkcs12 -in /path/to/original/key.p12 -passin pass:notasecret -nodes -nocerts | openssl rsa -out /etc/salt/pkey.pem
-
+    openssl pkcs12 -in /home/USER/project-id-0375632f3fe4.p12 -passin pass:notasecret -nodes -nocerts | openssl rsa -out /etc/salt/pkey.pem
```

Comment:
TODO(@supertom): make this work with the JSON credentials file so users don't have to type this archaic openssl command.  Also, with a master deployed in GCE, it should be able to use libcloud's "internal" auth using metadata tokens.